background.js

```diff
@@ -1,8 +1,15 @@
-if (typeof ('chrome') === 'undefined') {
-	var chrome = browser;
-}
-
-var browser = browser || chrome;
+import { chrome, browser } from './globals';
+import trackerData from './whotracksme.json';
```

Comment:
We can't use the the tracker database directly. For example, we need information like google-analytics.com belongs to Google. The tracker database only gives the info, google-analytics.

Reply:
@konarkmodi Do you know which data we need as a source to re-create the file required in `background.js`? Maybe this could be generated on `postinstall` hook automatically with a script hosted in the repository?

Reply:
@remusao : 

This is what we might need to adapt in postinstall: 
https://gist.github.com/konarkmodi/64a6eb9748041c502ebf528b0e858dc0.

Looking at the code in `background.js`, we only need a mapping of `host:company_name`
https://github.com/cliqz-oss/local-sheriff/blob/master/background.js#L202

Other place trackerData is used, is not needed so it can be removed:
Removing `https://github.com/cliqz-oss/local-sheriff/blob/master/background.js#L190` , 
`https://github.com/cliqz-oss/local-sheriff/blob/master/background.js#L202` and loading
`import company_hostname from './whotracksme.json';` based on gist should suffice.

Reply:
@konarkmodi Sorry for the delay, I've added the updating script + adapted the code as suggested. Thanks!

Reply:
@remusao : Nothing to be sorry about. I really appreciate your time on this. 
I will try and get this PR in this week.
Thanks a lot.